securedrop/journalist_app/admin.py

```diff
@@ -33,9 +35,30 @@ def manage_config():
             f = form.logo.data
             static_filepath = os.path.join(config.SECUREDROP_ROOT,
                                            "static/i/logo.png")
+            temp_static_filepath = os.path.join(config.SECUREDROP_ROOT,
+                                                "static/i/temp_logo.png")
+            with open(static_filepath) as img_file:
```

Comment:
As noted in other comments, I don't think we need to read this in. `os.rename` should just do the right thing on its own.